js/src/apbct-public-bundle.js

```diff
@@ -1600,6 +1600,133 @@ function apbctCancelAutocomplete(element) {
     }));
 }
 
+if (ctPublic.data__key_is_ok) {
+    if (document.readyState !== 'loading') {
+        apbctForceProtect();
```

Comment:
Do we need run this independent of the surrent setting statement? Maybe localize 'forms__force_protection' to ctPublic? I can't see any ways to skip encode/decode on public JS.

Reply:
[https://github.com/CleanTalk/wordpress-antispam/pull/477/files#diff-322eb766244dd8d3a92d89fe40c02788fcc2d870af63f6cb4deb27526fa1bc57R193](url)
Here it is checked whether the option is enabled, if not, the wrapper will not load, therefore there will be no request